docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -34,72 +47,58 @@ To build an APB:
 make apb_build
 ....
 
-Every APB has a Docker Hub repository set up for hosting images.
-When a PR for an APB repository is merged to master, the Docker Hub respository detects this and kicks off an automated build.
-If the build is successful, the resulting image is tagged as `latest`, replacing the previous `latest` image for that APB.
+Alternatively, if you have set up the alias described above you can directly use `apb build`.
 
-=== Setting up an Automated Build for APB Images
+=== Push an APB to a Local Openshift Cluster
 
-IMPORTANT: Docker repositories for APBs need to be created as an 'Automated Build' rather than a 'Repository'. There doesn't seem to be a way to add an automated build to a repository afterwards.
+Assuming your OpenShift cluster is up and running, you can push the APB image to the local OpenShift Docker Registry with:
 
-While signed into Docker Hub from a browser:
+....
+apb push -o
```

Comment:
`-o` may be optional here.

Reply:
Yes this seems to be brand new functionality so I will have to test it out and see if it's available now

Reply:
`-o` was removed by https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/pull/187, so it'll disappear soon from the apb cli docker image